.github/workflows/docs.yml

```
@@ -0,0 +1,32 @@
+name: Publish docs
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 1
+      matrix:
+        os:
+        - ubuntu-latest
+        python-version:
+        - '3.12'
```

portray (の依存している hug) は Python 3.12 での `distutils` モジュール削除により動かなくなっているので、 `3.11` にすると良さそうです。
```suggestion
        - '3.11'
```